From dc5278a7a15da39b0ba908b8ded4ef7af7f3872d Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Thu, 22 Aug 2019 16:22:56 -0700
Subject: [PATCH] joyent/triton-dehydrated#20 dehydrated dependency sometimes
 mis-handles account ID Reviewed by: Phillip Steinbachs <undefined> Approved
 by: Phillip Steinbachs <undefined>

---
 deps/dehydrated | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/dehydrated b/deps/dehydrated
index fba49ba..0182549 160000
--- a/deps/dehydrated
+++ b/deps/dehydrated
@@ -1 +1 @@
-Subproject commit fba49ba28eb746eae127aa36e7f515beeaf0bbae
+Subproject commit 018254974cb6cbf93ba9249b06b2d923e37a9e88
-- 
2.21.0

